odm2api/ODM2/services/readService.py

```diff
@@ -549,19 +549,19 @@ def getAffiliations(self, ids=None, personfirst=None, personlast=None, orgcode=N
         all Affiliation objects in the database will be returned.
 
         Args:
-            ids (:obj:`list`, optional): List of AffiliationIDs. Defaults to None.
-            personfirst (:obj:`str`, optional): Person First Name. Defaults to None.
-            personlast (:obj:`str`, optional): Person Last Name. Defaults to None.
-            orgcode (:obj:`str`, optional): Organization Code. Defaults to None.
+            ids (list, optional): List of AffiliationIDs. Defaults to None.
+            personfirst (str, optional): Person First Name. Defaults to None.
+            personlast (str, optional): Person Last Name. Defaults to None.
+            orgcode (str, optional): Organization Code. Defaults to None.
 
         Returns:
             list: List of Affiliation objects
 
         Examples:
-            >>> read.getAffiliations(ids=[39,40])
-            >>> read.getAffiliations(personfirst='John',
+            >>> ReadODM2.getAffiliations(ids=[39,40])
+            >>> ReadODM2.getAffiliations(personfirst='John',
             ...                      personlast='Smith')
-            >>> read.getAffiliations(orgcode='Acme')
+            >>> ReadODM2.getAffiliations(orgcode='Acme')
```

Comment:
Is it helpful to have the module (`ReadODM2`) shown in the examples?? Seems like visual clutter to me. 

This is a general comment, not specific to `getAffiliations`

Reply:
> Is it helpful to have the module (`ReadODM2`) shown in the examples?? Seems like visual clutter to me. 

`ReadODM2` is the parent class of this method, right? I guessed that `read` was an instance of that class. If `getAffiliations`  is a static method it should be called with `ReadODM2().getAffiliations`, no? If not I guess that the confusing part is naming the instance the same name as the parent class, but I am OK with the examples.

Reply:
@emiliom, @ocefpaf I am following what pycharm is telling me. I get unresolved reference warning.. I can change to whatever.

> If getAffiliations is a static method it should be called with ReadODM2().getAffiliations, no?

getAffiliations is not a static method.

Here's an example of the usage: 
- https://github.com/ODM2/ODM2PythonAPI/blob/master/Examples/Sample.py#L38. `ReadODM2` get instantiated and then the functions are used.

Reply:
> I am following what pycharm is telling me

That is b/c pycharm is probably trying to lint that fragment of code. The original example probably omitted the instantiation of the class.

```python
read = ReadODM2()
```

Reply:
So keep `ReadODM2`?

Reply:
> So keep ReadODM2?

Yes.

Reply:
> So keep `ReadODM2`?

If that is a class method, yes. I need to check the code but it does not look like it.
See https://stackoverflow.com/questions/17134653/difference-between-class-and-instance-methods

If that is an instance method than you need to add more to the example, like:

```
read = ReadODM2()
read.<method>
```

Reply:
My executive decision: keep it for now, so we can move forward with the PR. We can discuss again later, elsewhere, to lay out general conventions.

Reply:
Like I suspect it is not a class method. Without an instance this cannot be called:

Wrong:

```python
from odm2api.ODM2.services import ReadODM2
ReadODM2.getAffiliations(ids=[39,40])
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-82d16725b458> in <module>()
----> 1 ReadODM2.getAffiliations(ids=[39,40])

TypeError: unbound method getAffiliations() must be called with ReadODM2 instance as first argument (got nothing instead)
```

Mocked "correct" example:
```python
r = ReadODM2(session)
r.getAffiliations(ids=[39,40])
<returns ids from session>
```

Reply:
Thanks for the follow-up, @ocefpaf. @lsetiawan, let's start compiling elsewhere a list of conventions we'll follow.